index.js

```diff
@@ -1,26 +1,165 @@
 const core = require('@actions/core');
 const github = require('@actions/github');
+const list = require('markdown-list');
+
+export function formatChType(type) {
```

Comment:
(type) = `(chore)` or `(feature)` or `(feat)` or `(bug)`...so it just returns the type if it's not null (where it turns it into `other` and in case we pass `feat` or `feature`, it just turns it into `feature` always.

Reply:
Interesting about `feat`, when does that happen?

Reply:
I've seen it before we started using the automated chType github workflow haha - anyway, it'll be fine even if that never happens.

Reply:
Just a nit, this is sort of a clever function, although I'm more of a fan of being explicit.

```ts
formatChType(type) {
   	const formats = {
		'feat': 'feature',
		'feature': 'feature',
		'chore': 'chore',
		'bug': 'bug',
		'other': 'other',
	}

	return formats[type] || formats.other;
}
```

I think this reads a little more clearly and I'm pretty sure is more performant.

Reply:
ah, i missed the `^` here, important detail. when would we get a non alphabetical character though? there are only 3 static types that come from clubhouse.